dev-perl/alien_build/alien_build-2.84.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Build external dependencies for use in CPAN"
+DESCRIPTION="This module provides tools for building external (non-CPAN) dependencies for CPAN. \
+It is mainly designed to be used at install time of a CPAN client, and work closely with \
+Alien::Base which is used at runtime."
+HOMEPAGE="https://metacpan.org/pod/Alien::Build"
+COPYRIGHT="2011-2022 by Graham Ollis"
+LICENSE="Artistic"
+REVISION="1"
+SOURCE_URI="https://cpan.metacpan.org/authors/id/P/PL/PLICEASE/Alien-Build-$portVersion.tar.gz"
+CHECKSUM_SHA256="8e891fd3acbac39dd8fdc01376b9abff931e625be41e0910ca30ad59363b4477"
+SOURCE_DIR="Alien-Build-$portVersion"
+#PATCHES="alien_sdl-$portVersion.patchset"
```

Comment:
This is leftover from copy-pasting recipes. It could well be that we need it though (with the correct name).